lib/updateVersion.js

```diff
@@ -803,7 +803,8 @@ function moveByteStrideToBufferView(gltf) {
 }
 
 function requireAccessorMinMax(gltf) {
```

Comment:
Might want to just clarify the helper function name here by specifying position.